packages/core-components/src/components/WarningPanel/WarningPanel.tsx

```diff
@@ -69,6 +69,9 @@ export type WarningPanelClassKey =
 
 const useStyles = makeStyles<BackstageTheme>(
   theme => ({
+    content: {
+      lineBreak: 'anywhere',
+    },
```

Comment:
Alright with me. Want to point out that an alternative could have been a `word-break: break-all;` in the existing `.message` class :)